openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -316,13 +319,16 @@ def process_request(self, request):
             # return the response.
             return process_request_response
 
+        # Note: request.session.get(SESSION_KEY) and request.session.session_key are different things. The former
+        #   contains the session user, the latter is the session id
```

Comment:
Thanks for this. I had looked this up with Tim and knew this at one point, and then forgot, because it is certainly not intuitive.